README.md

```diff
@@ -132,87 +135,87 @@ unmockedCommands: [string]
 For convenience you can specify commands as a string directly. So this
 
 ``` yaml
-protocol:
+steps:
   - command: git add .
   - command: git push
 ```
 
 can be written as
 
 ``` yaml
-protocol:
+steps:
   - git add .
   - git push
 ```
 
-#### Multiple protocols
+#### Multiple tests
 
-Multiple protocols can be specified using a YAML array:
+Multiple tests can be specified using a YAML array:
 
 ``` yaml
 # when given the 'push' argument, it pushes to the remote
 - arguments: push
-  protocol:
+  steps:
     - git add .
     - git push
 # when given the 'pull' argument, it just pulls
 - arguments: push
-  protocol:
+  steps:
     - git pull
 ```
 
-You can also put everything into a `protocols` field:
+You can also put everything into a `tests` field:
 
 ``` yaml
-protocols:
+tests:
   # when given the 'push' argument, it pushes to the remote
   - arguments: push
-    protocol:
+    steps:
       - git add .
       - git push
   # when given the 'pull' argument, it just pulls
   - arguments: push
-    protocol:
+    steps:
       - git pull
 ```
 
-## Recording protocols
+## Recording tests
 
-There is **experimental** support for recording protocols. You can either record
-protocols by passing in the `--record` command line flag, or you can put
-so-called holes into your protocols:
+There is **experimental** support for recording tests. You can either record
```

Comment:
it just occurred to me that this idea might need a bit more explanation. i don't believe there are other similar tools out there that can automatically generate tests for you, so the idea of "recording tests" isn't obvious IMO.

Reply:
Yeah, that's right. This could probably use recording snapshot tests as an analogy.

Do you think that this PR makes the readme worse? If not I would defer your concern to a different issue/PR.

Reply:
no i think the change is fine, but using a more common word made it clearer to me that this was confusing. yes, i would defer to a separate PR. i can make a ticket for it.